### DISCUSSION TITLE
"Position Not Available" when using Bed Tramming

### DISCUSSION THREAD

**Discussion by severandynasty** [mriscoc/Ender3V2S1#40] [Q&A] [answered]
Hello! I just switched over to your firmware last night and so far it looks good, but when using the Bed Tramming feature with a Creality brand BLTouch (-44, -6 offsets) the printer will not measure the right side positions because it thinks it is outside bounds.  I see you have the print bed size set to 230x230 and a probe safety margin of 25.  I'm not quite sure why its refusing to move the point because I know physically it will.  What setting do I need to change to get this to work?

**Comment by mriscoc** [comment, marked as answer]
Could you please write the version and parameters that you use? I will try to replicate your issue. Please load default settings and save them to the EEPROM could be a problem with values coming from old firmware.

Try increasing your probe margin

**Comment by severandynasty** [comment]
I was running on Marlin 2.0.9.3 before with the JyersUI DWIN_Set and a handful of changes to the configs.  Thanks for the quick resonse!

Edit: I take that back, when I adjust my probe offsets to the correct values it errors again.

Changes:
Rest printer to defaults and rebooted.
Updated probe offsets to -44, -6
Updated load and unload lengths for runout sensor and set to HIGH
Updated e steps to match my extruder

All other settings are default.  Additionally, when I did tram the bed I got all z values to 0.00 but the mesh shows fairly high deviation is two of the corners.

**Reply by mriscoc** [reply]
From https://github.com/mriscoc/Ender3V2S1/wiki/Bed-tramming-menu
For a position to be reachable (in X axis) must meet:
> Max X position + Probe X Offset  **>** Bed X size - Probe Margin 

Max X position (248) + Probe X Offset (-44) **>** Bed X size (230) - Probe Margin (25)
but
204 is less than 205

Increase your probe margin.

**Comment by severandynasty** [comment]
That got it, I didn't RTFM too close and assumed I needed to _down_ on the probe margin.  Thanks again!

**Comment by grrmisfit** [comment]
reset printer ,restored defaults, and I have same issue on S1. cannot probe back corners.

edit: had to change the probe  Y offset from 40.5 to 40

**Reply by severandynasty** [reply]
Well rather than change the offset, which will trick your printer into miscalculating the bed mesh point in relation to the nozzle, do as I did above based om @mriscoc suggestion and _increase_ the probe margin to allow for more room.

I would also suggest that the Probe Margin be set a little bit higher to accommodate the factory Creality BLTouch/CRTouch mounts.  In the V2 this is -44, -6.
